checklists/buddy.json

```diff
@@ -33,25 +33,25 @@
 	},
 	"greetThem": { 
 		"displayName": "Greet your buddy",
-		"description": "Today will be intense for them, so say hi and offer to answer any questions they have. The time they leave orientation varies based on their location. D.C. gets out around 12PM ET; NY 9AM ET; Chicago 4:30 PM CT; and SF 9AM PT.", 
+		"description": "Today will be intense for them, so say hi and offer to answer any questions they have. The time they leave orientation varies based on their location. D.C. gets out around 12PM ET; NY 9AM ET; Chicago 3:30 PM CT; and SF 9AM PT.", 
 		"daysToComplete": 0, 
 		"dependsOn": ["dayZero"]
 	},
 	"allHands": { 
 		"displayName": "All hands",
-		"description": "Help your buddy find the all hands meeting on Tuesday.", 
+		"description": "Help your buddy attend the all hands meeting on Tuesday.  Leah Gitter will add the new hire to the calendar invite.”, 
```

Comment:
Eeep, looks like one of those special quotation marks slipped in at the end of this item.